modules/nw-metallb-operator-setting-runtimeclass.adoc

```diff
@@ -5,7 +5,7 @@
 [id="nw-metallb-operator-setting-runtimeclass_{context}"]
 = Configuring a container runtime class in a MetalLB deployment
 
-You can optionally assign a container runtime class to `controller` and `speaker` pods by configuring the `MetalLB` custom resource. For example, for Windows workloads, you can assign a Windows runtime class to the pod, which uses this runtime class for all containers in the pod.  
+You can optionally assign a container runtime class to `controller` and `speaker` pods by configuring the `MetalLB` custom resource. For example, for Windows workloads, you can assign a Windows runtime class to the pod, which uses this runtime class for all containers in the pod.
```

Comment:
Does MetalLB need to be highlighted here? Are we referring to a specific component or just a general reference to MetalLB?

For example, `MetalLB` customer resource makes sense.